server/src/auth.js

```diff
@@ -860,3 +861,37 @@ function checkIPaddress(req, ip, cred) {
 	if (req.ip != ip && req.ips?.[0] != ip && req.connection?.remoteAddress != ip)
 		throw `Your connection has changed, please refresh your page or sign in again.`
 }
+
+authApi.restrictFilterToAuthorizedValues = function (req, ds, filter, term) {
+	let tvslst
+	if (filter) {
+		tvslst = filter
+		tvslst.join = 'and'
```

Comment:
The original filter `join` should not be changed from `or` to `and`. I'll fix this in my branch.